x/nexus/keeper/lockable_asset.go

```diff
@@ -176,11 +172,13 @@ func mint(ctx sdk.Context, bank types.BankKeeper, toAddr sdk.AccAddress, coin sd
 	return nil
 }
 
-func getCoinType(ctx sdk.Context, nexus types.Nexus, denom string) (types.CoinType, error) {
+func getCoinType(ctx sdk.Context, nexus types.Nexus, ibc types.IBCKeeper, denom string) (types.CoinType, error) {
 	switch {
 	// check if the format of token denomination is 'ibc/{hash}'
 	case isIBCDenom(denom):
 		return types.ICS20, nil
+	case isFromExternalCosmosChain(ctx, nexus, ibc, denom):
+		return types.ICS20, nil
```

Comment:
why distinguish these two?

Reply:
Check the previous implementation of `prepareTransfer` in the axelarnet msg_server.go https://github.com/axelarnetwork/axelar-core/pull/2186/files#